common-healing.lic

```diff
@@ -243,8 +243,8 @@ module DRCH
   def bind_wound(part, person = 'my')
     snap = [DRC.left_hand, DRC.right_hand]
     if /You \w+ remove/ =~ DRC.bput("tend #{person} #{part}", 'You work carefully at tending', 'That area has already been tended to', 'That area is not bleeding', 'You fumble', 'too injured for you to do that', 'You \w+ remove')
-      DRC.bput("drop my #{DRC.left_hand}", 'You drop', 'What were you') if DRC.left_hand != snap.first
-      DRC.bput("drop my #{DRC.right_hand}", 'You drop', 'What were you') if DRC.right_hand != snap.last
+      DRCI.dispose(DRC.left_hand) if DRC.left_hand != snap.first
```

Comment:
❌ `common-items` needs to be added to line 5

Reply:
Added.